### PR TITLE
Use Example URL as fullUrl

### LINKED ISSUE #35: Use Example URL as fullUrl
The UUID is no absolute URI as required by the spec. We should use a RESTful URL of an example server here.

### PR DESCRIPTION
**Pull request by noemide**
Fixes #35

### CODE REVIEW

**Review by alexanderkiel — CHANGES_REQUESTED** [review]
We don't need the absolute URL in references.

**Comment by noemide** [comment]
I'm not 100% sure about the reference to Organization... Is it resolved to [base]/Organisation/2 automatically?

**Comment by alexanderkiel** [comment]
> I'm not 100% sure about the reference to Organization... Is it resolved to [base]/Organisation/2 automatically?

No not according to https://www.hl7.org/fhir/bundle.html#references. Here I think we need an absolute URL referring to Blaze itself. There is [this](https://www.hl7.org/fhir/references.html#literal) saying that a relative URL is normally relative to the Service Base URL but not in a Bundle. 

Can you please read this? https://www.hl7.org/fhir/managing.html I haven't read it yet. But it sounds good.

**Comment by noemide** [comment]
> 
> 
> > I'm not 100% sure about the reference to Organization... Is it resolved to [base]/Organisation/2 automatically?
> 
> No not according to https://www.hl7.org/fhir/bundle.html#references. Here I think we need an absolute URL referring to Blaze itself. There is [this](https://www.hl7.org/fhir/references.html#literal) saying that a relative URL is normally relative to the Service Base URL but not in a Bundle.
> 
> Can you please read this? https://www.hl7.org/fhir/managing.html I haven't read it yet. But it sounds good.

I decided to go with a full reference to the already existing Collection, prefixed it with blaze to it is clear that this must be an already existing FHIR resource on a server. This seems to be correct according to the "How to resolve a refernce in a Bundle" section.

File: examples/transaction-0.json [hunk 1]
```diff
@@ -223,7 +223,7 @@
           {
             "url": "https://fhir.bbmri.de/StructureDefinition/Custodian",
             "valueReference": {
-              "reference": "Organization/collection-2"
+              "reference": "http://blaze.example.com/Organization/collection-2"
```
Comment (alexanderkiel) [inline examples/transaction-0.json]: here I would append the context path `/fhir` because Blaze works with it.

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: examples/transaction-0.json (hunk 1 above)
Comment (noemide) [reply examples/transaction-0.json]: Done. I also added some Ducumentation that uses /fhir in its examples.

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**